### PR TITLE
Ruhl/add dash client secret

### PR DESCRIPTION
**Pull request by robert-uhl**

### CODE REVIEW

**Review by armaanv — APPROVED** [review]
LGTM

File: etc/helm/pachyderm/templates/_helpers.tpl [hunk 1]
```diff
@@ -19,3 +19,11 @@ LOCAL
 {{ fail "pachd.storage.backend required when no matching deploy target found" }}
 {{- end -}}
 {{- end -}}
+
+{{- define "pachyderm.dashSecret" -}}
```
Comment (chainlink) [inline etc/helm/pachyderm/templates/_helpers.tpl:23]: can simplify with `default`

**Review by chainlink — CHANGES_REQUESTED** [review]
I think we can eliminate the need for the template by using `default` instead

File: etc/helm/pachyderm/templates/_helpers.tpl (hunk 1 above)
Comment (robert-uhl) [reply etc/helm/pachyderm/templates/_helpers.tpl:23]: And done.

**Review by chainlink — APPROVED** [review]

**Pull request merged**